.github/scripts/chromium_i686_common.sh

```
@@ -1931,17 +2031,61 @@ run_build_until_checkpoint() {
+    python3 "${WORKSPACE}/scripts/ninja_stall_watchdog.py" \
+      --stall-seconds "$((ninja_stall_minutes * 60))" \
+      --timeout-seconds "${remaining}" \
+      2>&1 | tee -a "${BUILD_LOG}"
     status=${PIPESTATUS[0]}
     set -o pipefail
     set -e
     tail -c "+$((pass_log_start + 1))" "${BUILD_LOG}" > "${pass_log}"
 
+    if [ -s "${stall_marker}" ] && [ -s "${watchdog_error_marker}" ]; then
+      rm -f -- "${stall_marker}" "${watchdog_error_marker}"
+      echo "no_progress_streak=${prior_no_progress_streak}" >> "${output_file}"
```

Shellcheck (0.11.0)

[style] Consider using { cmd1; cmd2; } >> file instead of individual redirects.

(SC2129)